modules/mod_articles/tmpl/default_items.php

```diff
@@ -44,6 +48,18 @@
                             </<?php echo $item_heading; ?>>
                         <?php endif; ?>
 
+                        <?php if ($item->state == 0) : ?>
```

Comment:
```suggestion
                        <?php if ($item->state === 0) : ?>
```

state should be an integer if not we should cast it. after this I think some tests are needed than we can merge it.